packages/layout/src/Grid.ts

```diff
@@ -497,11 +504,32 @@ export class Grid extends HTMLWidget {
     postSelectionChange() {
     }
 
+    applyLayout(layoutArr: ICellPosition[]) {
+        this.divItems.each((d, i) => {
+            if (layoutArr[i]) {
+                const [x, y, w, h] = layoutArr[i];
+                d
+                    .gridCol(x)
+                    .gridRow(y)
+                    .gridColSpan(w)
+                    .gridRowSpan(h)
```

Comment:
w + h can be undefined as per your typing above?

Reply:
Hmm... I guess they shouldn't be optional